rest/src/test/java/org/jboss/pnc/reqour/rest/endpoints/TranslationEndpointIT.java

```diff
@@ -21,10 +22,12 @@
 import org.junit.jupiter.api.Test;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.jboss.pnc.reqour.rest.endpoints.TestConstants.TEST_USER;
 
 @QuarkusTest
 @TestHTTPEndpoint(TranslateEndpoint.class)
 @TestProfile(TranslationProfile.class)
+@TestSecurity(user = TEST_USER, roles = { OidcRoleConstants.PNC_APP_REPOUR_USER })
```

Comment:
Same here, We don't need authentication/authorization for the translate endpoint